tests/test_plugins/fixtures/dollar_math.md

```diff
@@ -300,6 +300,17 @@ $$a+b$$
 </section>
 .
 
+equation following text. (valid=False)
+.
+Here is some text
+$$a+b$$
+.
+<p>Here is some text</p>
```

Comment:
@chrisjsewell should this `xml` markup be:

- [ ] what is currently produced , or
- [ ] the syntax that should be produced